src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.cs

```diff
@@ -9,24 +9,24 @@ namespace System.Diagnostics
 {
     public partial class DiagnosticListener : System.Diagnostics.DiagnosticSource, System.IDisposable, System.IObservable<System.Collections.Generic.KeyValuePair<string, object>>
     {
-        public DiagnosticListener(string name) { }
+        public DiagnosticListener(string? name) { }
         public static System.IObservable<System.Diagnostics.DiagnosticListener> AllListeners { get { throw null; } }
-        public string Name { get { throw null; } }
+        public string? Name { get { throw null; } }
         public virtual void Dispose() { }
         public bool IsEnabled() { throw null; }
         public override bool IsEnabled(string name) { throw null; }
-        public override bool IsEnabled(string name, object arg1, object arg2 = null) { throw null; }
+        public override bool IsEnabled(string name, object? arg1, object? arg2 = null) { throw null; }
         public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer) { throw null; }
-        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer, System.Func<string, object, object, bool> isEnabled) { throw null; }
-        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer, System.Predicate<string> isEnabled) { throw null; }
-        public override string ToString() { throw null; }
+        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer, System.Func<string, object?, object?, bool>? isEnabled) { throw null; }
+        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer, System.Predicate<string>? isEnabled) { throw null; }
+        public override string? ToString() { throw null; }
         public override void Write(string name, object value) { }
```

Comment:
I think `value` should be nullable.

Reply:
I actually update both to nullable according to @safern 's comment and there is nothing preventing from that, haven't had udpated ref assemblies yet, jsut did few minutes ago